### PR TITLE
attach signature in order message

### PR DESCRIPTION
**Pull request by HAOYUatHZ**

### CODE REVIEW

File: src/msg/msg_utils.rs [hunk 1]
```diff
@@ -50,6 +50,30 @@ impl From<String> for TokenIdPair {
     }
 }
 
+// TODO:
+fn hash_order(_order: &crate::types::matchengine::messages::Order) -> String {
```
Comment (lispc) [inline src/msg/msg_utils.rs:54]: why return String here rather than BigInt/Fr?

File: src/msg/msg_utils.rs [hunk 2]
```diff
@@ -50,6 +50,30 @@ impl From<String> for TokenIdPair {
...
+    unimplemented!()
+}
+
+use crate::account::SignatureBJJ;
+use std::convert::TryInto;
+impl From<&matchengine::messages::Order> for crate::account::Signature {
```
Comment (lispc) [inline src/msg/msg_utils.rs:60]: I think Order and Signature are two different things. 'From/Into' seems not to be good name. pub fn extractSignature(&Order) ?

File: src/msg/msg_utils.rs (hunk 1 above)
Comment (HAOYUatHZ) [reply src/msg/msg_utils.rs]: fixed in https://github.com/Fluidex/rollup-state-manager/pull/80/commits/727648de2cf9c372477f870fa95690acda3848e6

File: src/msg/msg_utils.rs (hunk 2 above)
Comment (HAOYUatHZ) [reply src/msg/msg_utils.rs]: https://github.com/Fluidex/rollup-state-manager/pull/80/commits/727648de2cf9c372477f870fa95690acda3848e6

**Review by lispc — APPROVED** [review]

**Pull request merged**